pinot-core/src/test/java/org/apache/pinot/core/data/manager/BaseTableDataManagerTest.java

```diff
@@ -614,6 +622,61 @@ public void testDownloadAndDecrypt()
     }
   }
 
+  // case 2: if the attempt to download from deep storage exceeds, invoke downloadFromPeers.
+  @Test
+  public void testDownloadAndDecryptCase2() throws Exception {
```

Comment:
NIT: instead of Case2, be more specific in the test name: it is about downloading from peers.